plugins/ltac2/tac2env.ml

```diff
@@ -134,16 +134,24 @@ let define_notation kn tac =
 
 let interp_notation kn = KNmap.find kn !ltac_notations
 
+let ml_tactic_name ~plugin tactic = {
```

Comment:
If you want to merge a version of this that just adds the wrapper without changing behavior, then we can get a rewiter overlay merged and this change can be benched against the Ltac2 code in the rewriter and Fiat Crypto